Packs/Mattermost/Scripts/MattermostAskUser/MattermostAskUser.yml

```diff
@@ -32,11 +32,17 @@ args:
   - "false"
   description: Indicates whether to use one-time entitlement or a persistent one.
   defaultValue: "false"
+- defaultValue: Thank you **{user}**. You have answered **{response}** .
+  description: The reply to send to the user. Use the templates {user} and {response} to incorporate these in the reply. (i.e., "Thank you **{user}**. You have answered **{response}**.").
+  name: reply
+- defaultValue: 1 day
+  description: Time until the question expires. For example - 1 day. When it expires, a default response is sent. Default value is 1 day.
+  name: lifetime
+- description: Default response in case the question expires.
+  name: defaultResponse
```

Comment:
Why have we created this (and other) args with camelcase instead of snake case?

Reply:
Taken from SlackAskv2 script

Reply:
Changed it to fit standarts